aws_xray_sdk/core/utils/search_pattern.py

```diff
@@ -12,74 +12,47 @@ def wildcard_match(pattern, text, case_insensitive=True):
     if pattern is None or text is None:
         return False
 
-    pattern_len = len(pattern)
-    text_len = len(text)
-    if pattern_len == 0:
-        return text_len == 0
+    if len(pattern) == 0:
+        return len(text) == 0
 
     # Check the special case of a single * pattern, as it's common
     if pattern == '*':
         return True
 
-    if case_insensitive:
-        pattern = pattern.lower()
-        text = text.lower()
+    # If elif logic Checking different conditions like match between the first i chars in text
+    # and the first j chars in pattern, checking pattern has '?' or '*' also check for case_insensitivity
```

Comment:
there's no j; it should be a p

Reply:
Thanks for the correction! Committed that change.